<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<comment> Copyright (C) 2019 - 2021 by Pedro Mendes, Rector and Visitors of the </comment>
<comment> University of Virginia, University of Heidelberg, and University </comment>
<comment> of Connecticut School of Medicine. </comment>
<comment> All rights reserved. </comment>

<comment> Copyright (C) 2017 - 2018 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc., University of Heidelberg, and University of </comment>
<comment> of Connecticut School of Medicine. </comment>
<comment> All rights reserved. </comment>

<comment> Copyright (C) 2010 - 2016 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc., University of Heidelberg, and The University </comment>
<comment> of Manchester. </comment>
<comment> All rights reserved. </comment>

<comment> Copyright (C) 2009 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc., EML Research, gGmbH, University of Heidelberg, </comment>
<comment> and The University of Manchester. </comment>
<comment> All rights reserved. </comment>

 <comment> All rights reserved. </comment>
 <class>CQReactionsWidget</class>
 <widget class="CopasiWidget" name="CQReactionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>456</height>
   </rect>
  </property>
  <property name="caption" stdset="0">
   <string>Reactions</string>
  </property>
  <layout class="QGridLayout">
   <item row="8" column="0" colspan="2">
    <layout class="QHBoxLayout">
     <item>
      <spacer name="horizontalSpacerLeft">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mpBtnNew">
       <property name="text">
        <string>&amp;New</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mpBtnDelete">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>&amp;Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mpBtnClear">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Delete &amp;All</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerRight">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="CQTableView" name="mpTblReactions">
     <property name="editTriggers">
      <set>QAbstractItemView::CurrentChanged|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" rowspan="3" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Search:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mpLEFilter"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CopasiWidget</class>
   <extends>QWidget</extends>
   <header>copasi/UI/copasiWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CQTableView</class>
   <extends>QTableView</extends>
   <header>copasi/UI/CQTableView.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mpTblReactions</tabstop>
  <tabstop>mpBtnDelete</tabstop>
  <tabstop>mpBtnClear</tabstop>
 </tabstops>
 <includes>
  <include location="local">copasi/UI/copasiWidget.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>mpBtnClear</sender>
   <signal>pressed()</signal>
   <receiver>CQReactionsWidget</receiver>
   <slot>slotBtnClearClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>443</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpBtnDelete</sender>
   <signal>pressed()</signal>
   <receiver>CQReactionsWidget</receiver>
   <slot>slotBtnDeleteClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>443</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpTblReactions</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>CQReactionsWidget</receiver>
   <slot>slotDoubleClicked(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpBtnNew</sender>
   <signal>pressed()</signal>
   <receiver>CQReactionsWidget</receiver>
   <slot>slotBtnNewClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
 </connections>
  <slots>
    <slot>slotBtnNewClicked()</slot>
    <slot>slotBtnDeleteClicked()</slot>
    <slot>slotBtnClearClicked()</slot>
    <slot>slotSelectionChanged()</slot>
    <slot>slotDoubleClicked(QModelIndex)</slot>
    <slot>dataChanged(QModelIndex, QModelIndex)</slot>
    <slot>slotFilterChanged()</slot>
  </slots>
</ui>
